tox.ini

```diff
@@ -11,4 +11,4 @@ commands = coverage run runtests.py
 deps =
     django18: Django>=1.8,<1.9
     django110: Django>=1.10,<1.11
-    -r{toxinidir}/requirements-test.txt
+    -rrequirements-test.txt
```

Comment:
more cleanup! python 3.2 wasnt actually being tested on travis, and for whatever reason the requirements weren't updating